packages/react-component-library/src/components/TopLevelNavigation/Sidebar/Sidebar.stories.tsx

```diff
@@ -89,18 +95,8 @@ export const Default: ComponentStory<typeof Sidebar> = (props) => {
 
   return (
     <SidebarWrapper>
-      <div style={{ maxHeight: '30rem' }}>
```

Comment:
These intermediary divs were redundant as `StyledSidebar` has `max-height` applied to it already. Removing it did reveal a small problem with the `StyledSidebar` styles (see comment left there).